docs/data/tree-view/rich-tree-view/expansion/expansion.md

```diff
@@ -33,8 +33,33 @@ Use the `onItemExpansionToggle` prop if you want to react to an item expansion c
 
 {{"demo": "TrackItemExpansionToggle.js"}}
 
-## Change item expansion
+## Imperative API
 
-You can use the `setItemExpansion` API method to imperatively change the expansion of an item:
+:::success
+To use the `apiRef` object, you need to initialize it using the `useTreeViewApiRef` hook as follows:
 
-{{"demo": "ChangeItemExpansion.js"}}
+```tsx
+const apiRef = useTreeViewApiRef();
+
+return <RichTreeView apiRef={apiRef} items={ITEMS}>;
+```
+
+`apiRef` will be undefined during the first render and will then contain methods allowing you to imperatively interact with the Tree View.
```

Comment:
```suggestion
`apiRef` will be `undefined` during the first render. When initialised it will contain methods allowing you to imperatively interact with the Tree View.
```

Reply:
I think that your version looses the idea that after the 1st render it's good, the "When initialized" does not say that after 1st render it's good

Reply:
Yeah, makes sense. 
Maybe something like this?

```suggestion
When your component first renders, `apiRef` will be `undefined`. Once the component is fully initialized, `apiRef` holds methods for interacting with the Tree View directly.
```

Reply:
but feel free to ignore ... 👍🏼

Reply:
`Once the component is fully initialized` is still something that might be longer than the 1st render IMHO.

How about:

```suggestion
When your component first renders, `apiRef` will be `undefined`. After this initial render,, `apiRef` holds methods to interact imperatively with the Tree View.
```

I also kept "imperatively" because we use it a lot